### PR TITLE
Fixed Crash Bundle ID

### PR DESCRIPTION
**Pull request by Eliottiosdevs**
Fixed a crash with force unwrap Bundle ID that caused crash #42

### CODE REVIEW

File: ReviewMonitor.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -283,9 +283,9 @@
 				TargetAttributes = {
 					693F5FA21F61F23F00FD8859 = {
 						CreatedOnToolsVersion = 8.3.2;
-						DevelopmentTeam = CEBY93VF85;
```
Comment (RishabhTayal) [inline ReviewMonitor.xcodeproj/project.pbxproj]: Let's not make any changes to the project code signing settings. Could you revert this file change?

File: ReviewMonitor/Model/App.swift [hunk 2]
```diff
@@ -16,7 +16,7 @@ class App: NSObject {
 
     init(dict: [String: Any]) {
         name = dict["name"] as! String
-        bundleId = dict["bundle_id"] as! String
+        bundleId = dict["bundle_id"] as? String ?? "-"
```
Comment (RishabhTayal) [inline ReviewMonitor/Model/App.swift]: Do you know in what case `bundle_id` will be `nil` on iTunes Connect? Isn't it a required field when you create an app there?

Comment (Eliottiosdevs) [reply ReviewMonitor/Model/App.swift]: When an app has no submitted builds, you can set the bundle ID to none. As I'm having now.

Comment (RishabhTayal) [reply ReviewMonitor/Model/App.swift]: Got it. Thanks 👍. Please revert the project file changes and then I will merge it.

File: ReviewMonitor.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Eliottiosdevs) [reply ReviewMonitor.xcodeproj/project.pbxproj]: Ok, Understand that ;). Will do

Comment (Eliottiosdevs) [reply ReviewMonitor.xcodeproj/project.pbxproj]: Should be OK. (I think)

**Comment by Eliottiosdevs** [comment]
Thanks!

**Comment by RishabhTayal** [comment]
Why was this PR closed? I haven't merged it yet. Don't close it if you want this to be merged.

**Comment by RishabhTayal** [comment]
Can you re-open it?

**Comment by Eliottiosdevs** [comment]
Sorry. I thought I was on another project...

**Comment by RishabhTayal** [comment]
Merged!

**Comment by Eliottiosdevs** [comment]
Thanks!

**Pull request merged**